syncstorage/storage/sql/dbconnect.py

```diff
@@ -112,19 +112,39 @@ def __new__(cls, *args, **kwds):
 PAYLOAD_TYPE = PAYLOAD_TYPE.with_variant(postgresql.TEXT(), 'postgresql')
 
 
-def _get_bso_columns(table_name):
+# Common column definitions between BSO and batch upload item tables
+
+def _common_columns(ttl):
     return (
-        Column("userid", Integer, primary_key=True, nullable=False,
-               autoincrement=False),
-        Column("collection", Integer, primary_key=True, nullable=False,
-               autoincrement=False),
-        Column("id", String(64), primary_key=True, autoincrement=False),
         Column("sortindex", Integer),
         Column("modified", BigInteger, nullable=False),
+        # MySQL no likey "DEFAULT ''".  Results in an error:
+        #     ERROR 1101 (42000): BLOB, TEXT, GEOMETRY or JSON column 'payload'
+        #                         can't have a default value
+        #
+        # By adding the following lines to my.cnf you can turn the fatal error
+        # into a warning:
+        #
+        # [mysqld]
+        # sql_mode=MYSQL40
+        #
         Column("payload", PAYLOAD_TYPE, nullable=False, server_default=""),
         Column("payload_size", Integer, nullable=False,
                server_default=sqltext("0")),
-        Column("ttl", Integer, server_default=sqltext(str(MAX_TTL))),
+        Column("ttl", Integer, server_default=sqltext(str(ttl)))
```

Comment:
All calls to this function pass in `MAX_TTL` as the arguments, we can probably simplify by just continuing to refer to it directly here.